recipes/opencolorio/all/conanfile.py

```diff
@@ -189,6 +186,20 @@ def package_info(self):
 
         self.cpp_info.libs = ["OpenColorIO"]
 
+        self.cpp_info.requires.append("expat::expat")
```

Comment:
These requirements are only necesary when some of them are optional, or where not the whole package is needed. If no requirements are specified, all available ones coming from the direct dependencies are used

So I'd say we can revert this bit, as that's what this code is doing, adding all available ones. We can let Conan handle it in this case :)